src/helpers/isPresent.js

```diff
@@ -0,0 +1,5 @@
+export default (input) => {
```

Comment:
What do you think about adding a quick and simple unit test for this?

Reply:
Development driven tests. It did cross my mind, yes. I'll pop one in.